### DISCUSSION TITLE
in the primeng documentation, when I click on stepper (on the left side menu), everything on the left side is collapsed

### DISCUSSION THREAD

**Discussion by BoisAuLit** [primefaces/community#1493] [PrimeNG]
in the primeng documentation, when I click on stepper (on the left side menu), EVERYTHING IS COLLAPSED (ACCORDIONS). Could you please fix this? If you don't know what I mean, 
![Screenshot 2024-03-22 at 00 49 08](https://github.com/primefaces/community/assets/18257978/6a04f455-ef20-47d1-8968-a7ff5aebf784)

**Comment by ThoSap** [comment]
I cannot reproduce this on https://primeng.org/ `v17.12.0`.

**Reply by BoisAuLit** [reply]
@ThoSap I can reproduce this issue on all the browsers (firefox, chrome, safari) on my operating system Mac OS X Sonoma, Macbook Pro m1 14 inch.

**Reply by ThoSap** [reply]
Thanks for clarifying what OS and browsers you are using. I tested on Windows and cannot test this on a macOS.
